src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -282,32 +284,41 @@ export const EnterInfo = ({
         </dl>
       </div>
       <div className={styles.stepButtonWrapper}>
-        <Button
+        <TelemetryButton
           variant="secondary"
           onPress={() => confirmDialogState.close()}
           className={styles.startButton}
+          event={{
+            module: "button",
+            name: "click",
+            data: {
+              button_id: "edit_free_scan",
```

Comment:
I think this is already handled in the `onOpenChange` handler on line 95, no?

(Though props for the eagle eye in checking that Tony wanted this event!)